src/Sylius/Bundle/CoreBundle/EventListener/UserDeleteListener.php

```diff
@@ -42,14 +42,14 @@ public function deleteUser(ResourceEvent $event)
     {
         $user = $event->getSubject();
 
-        if (!$user instanceof UserInterface) {
+        if (!$user instanceof GroupInterface) {
```

Comment:
This is wrong. Why you expect group as subject instead of user? You still use `user.pre_delete` event...